load-tests/smoke.js

```
@@ -0,0 +1,130 @@
+    { duration: "30s", target: 0 }, // ramp down
+  ],
+  thresholds: {
+    http_req_duration: ["p(95)<3000"],
+    errors: ["rate<0.05"],
+    // availability is the public booking hot path (rate-limited 30/min/IP —
+    // expect 429s to register as errors if a single-IP run exceeds that)
+    slot_availability_duration: ["p(95)<1500"],
+  },
+};
+
+const BASE_URL = __ENV.BASE_URL || "http://localhost:3000";
```

ESLint

[error] '__ENV' is not defined.

(no-undef)